improver_tests/calibration/ensemble_calibration/test_EstimateCoefficientsForEnsembleCalibration.py

```diff
@@ -892,15 +782,19 @@ def setUp(self):
         self.current_cycle = "20171110T0000Z"
         self.distribution = "gaussian"
 
-        self.coeff_names = ["gamma", "delta", "alpha", "beta"]
-        self.coeff_names_realizations = [
+        coeff_names = ["gamma", "delta", "alpha", "beta"]
+        self.coeff_names = [f"emos_coefficient_{s}" for s in coeff_names]
+        coeff_names_realizations = [
             "gamma",
             "delta",
             "alpha",
             "beta0",
             "beta1",
             "beta2",
         ]
+        self.coeff_names_realizations = [
+            f"emos_coefficient_{s}" for s in coeff_names_realizations
+        ]
```

Comment:
You have some duplication here, from above - would it be appropriate to define some of this stuff at the top of the file?  ("No", "not really" or "I don't think so" are acceptable answers here...)

Reply:
I've done a little bit of centralisation here, which I think removes this particular duplication.